routes/web.php

```diff
@@ -28,8 +28,9 @@
 Route::controller(LoginController::class)->group(function () {
     Route::get('/login/google', 'redirectToGoogle')->name('login.google');
     Route::get('/login/google/callback', 'handleGoogleCallback');
+    Route::get('login/facebook', 'redirectToFacebook')->name('auth.facebook');
```

Comment:
please change the route name as 'login.facebook' to be simillar to the google auth route